dev-client/src/components/sheets/OverlaySheet.tsx

```diff
@@ -50,7 +50,7 @@ export const BottomSheetModal = forwardRef<
       Header,
       children,
       trigger,
-      CloseButton,
+      Closer,
```

Comment:
question: why is this called Closer?

Reply:
There was a name collision with the CloseButton component.